### DISCUSSION TITLE
Registration/Partnership.

### DISCUSSION THREAD

**Discussion by Sadaf-KCITSC** [vercel/community#897] [Help] [answered]
Hello this is Sadaf Walizada Operation manager from kabul cell
Information Technologys
Services Company. We are looking
forward to getting a partnership with
Vercel company in Afghanistan. I want
to know how I can register our server
to your company. I will appreciate it if
I get a zoom call for sharing more
information. Thank you.

**Comment by amyegan** [comment, marked as answer]
Thanks for reaching out @Sadaf-KCITSC! For help customizing a [plan](https://vercel.com/pricing) to suit your needs, please visit [vercel.com/contact/sales](https://vercel.com/contact/sales). From there, you will be connected with a person who can walk you through the features and options that Vercel offers. For more information about partnerships, please visit [vercel.com/partners](https://vercel.com/partners)
